tablelog.py

```diff
@@ -0,0 +1,133 @@
+import argparse
+import copy
+import glob
+import json
+import logging
+import os
+import random
+import shutil
+import time
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import numpy as np
+import pytest
+import torch
+import tqdm
+import wandb
+from torch.utils.data import DataLoader
```

Comment:
can you change this to only import what you need, I don't think you're using some of this stuff (definitely not argparse or shutil or pytest)